crates/primitives/chain-id/src/lib.rs

```diff
@@ -0,0 +1,20 @@
+//! Traits for chain ids.
+#![cfg_attr(not(feature = "std"), no_std)]
+
+use mp_felt::Felt252Wrapper;
+
+/// ChainId for Starknet Goerli testnet
+pub const SN_GOERLI_CHAIN_ID: Felt252Wrapper = Felt252Wrapper(starknet_ff::FieldElement::from_mont([
+    3753493103916128178,
+    18446744073709548950,
+    18446744073709551615,
+    398700013197595345,
+]));
+
+/// ChainId for Starknet Mainnet
+pub const SN_MAIN_CHAIN_ID: Felt252Wrapper = Felt252Wrapper(starknet_ff::FieldElement::from_mont([
+    17696389056366564951,
+    18446744073709551615,
+    18446744073709551615,
+    502562008147966918,
+]));
```

Comment:
Can you add two tests where you assert that the const are equal to the value build with `from_hex_be(b"SN_GOERLI")` and `from_hex_be(b"SN_MAINET")`.

This way we are sure those numbers are really what they claim to be